about.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+$db = new PDO('mysql:host=127.0.0.1; dbname=osamasCMSDB', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$aboutMeArticleQuery = $db->prepare("SELECT `aboutMeArticle` FROM `aboutMe`;");
+$aboutMeArticleQuery->execute();
+$aboutMeArticle = $aboutMeArticleQuery->fetchAll();
+
+$contactIconQuery = $db->prepare("SELECT `contactIcon` FROM `footer`");
+$contactIconQuery->execute();
+$contactIcon = $contactIconQuery->fetchAll();
+
+$smallContactIconQuery = $db->prepare("SELECT `smallContactIcon` FROM `footer`");
+$smallContactIconQuery->execute();
+$smallContactIcon = $smallContactIconQuery->fetchAll();
+
+$contactEmailQuery = $db->prepare("SELECT `contactEmail` FROM `footer`");
+$contactEmailQuery->execute();
+$contactEmail = $contactEmailQuery->fetchAll();
+
+$emailSubjectQuery = $db->prepare("SELECT `emailSubject` FROM `footer`");
```

Comment:
each query should be a function in a diff file that is called here

Reply:
done.